### DISCUSSION TITLE
Wonderswan pinout

### DISCUSSION THREAD

**Discussion by Bandmo** [sanni/cartreader#340] [Need help]
@sanni So I am very new to building PCBs. I would like to build an adaptor for the wonderswan for version 4 cart reader. The pinout spreadsheet has the first pin as "A-1". What does this mean?

And I have been trying to follow your tutorial in the wiki to build my own adaptor, and I am getting really lost when going between the different apps and knowing how to do different features.

Is it possible that you would have a video showing the process? Thanks!

**Comment by sanni** [comment]
splash5 has made an adapter PCB for Wonderswan here: https://github.com/sanni/cartreader/pull/337 (order with 1.2mm thickness instead of the default 1.6mm or else it will not fit into the SNES slot)

**Comment by Bandmo** [comment]
THANK YOU! I have been racking my brain to find where he posted it. The
link that was on the original post sent me to nonsense. Thanks for what you
have done with the development of this project. I'm really enjoying it so
far.
